snippets/UUIDGenerator.md

```diff
@@ -4,13 +4,26 @@ Generates a UUID.
 
 Use `crypto` API to generate a UUID, compliant with [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) version 4.
 
+Please note that this version will only work in **Node.JS** environments!
+
 ```js
+const crypto = require("crypto");
```

Comment:
needed only in nodeJS, browsers expose a crypto object

Reply:
Therefore, this should be added as part of the description and not part of the code.